README.md

```diff
@@ -1 +1,94 @@
-# Capstone_project_1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HTML/CSS & JavaScript Capstone Project
+
+> We built an entire website from scratch containing two pages: the "home" and "about" pages. We based the design on a template provided by Creative Commons. Here we applied what we have learned in Module 1 of Microverse. Up to now, we implemented the mobile and desktop versions of this tiny website; also we added a dynamic menu for the mobile version and created dynamically one section of the main page using JavaScript. This will be part of the Portfolio projects that will be shown to recruiters.
+
+<br>
+
+![screenshot](./images/mobile_version.png)
+
+<br>
+
+![screenshot](./images/desktop_version.png)
+
+## Built With
+
+- HTML, CSS and JavaScript.
+- GitHub
+
+## Live Demo (if available)
+
+[Live Demo Link](https://jevazquezb.github.io/Capstone_project_1/)
+
+## Project documentation (if available)
```

Comment:
[Optional]
As you already have added the loom link for your capstone project, you may remove "(if available)" 😉